specification/src/main/asciidoc/platform/PlatformOverview.adoc

```diff
@@ -456,107 +453,107 @@ The Java™ Authentication and Authorization
-product. The Java™ Authentication Service Provider Interface for
-Containers (JASPIC) defines an SPI by which authentication providers
+custom authorization service providers to be plugged into any Jakarta EE
+product. Jakarta™ Authentication
+defines an SPI by which authentication providers
 implementing message authentication mechanisms may be integrated in
-client or server message processing containers or runtimes. The Java EE
-Security API leverages JASPIC, but provides an easier to use SPI for
+client or server message processing containers or runtimes. Jakarta
+Security leverages Jakarta Authentication, but provides an easier to use SPI for
 authentication of users of web applications and defines identity store
 APIs for authentication and authorization.
 
 === Web Services
 
-Java EE provides full support for both clients
+Jakarta EE provides full support for both clients
 of web services as well as web service endpoints. Several Java
-technologies work together to provide support for web services. The Java
-API for XML Web Services (JAX-WS) and the Java API for XML-based RPC
-(JAX-RPC) both provide support for web service calls using the SOAP/HTTP
+technologies work together to provide support for web services. Jakarta
+XML Web Services and Jakarta XML-based RPC
+both provide support for web service calls using the SOAP/HTTP
 protocol. JAX-WS, which is included in Java SE, is the primary API for
-web services and is a follow-on to JAX-RPC. JAX-WS offers extensive web
+web services and is a follow-on to Jakarta XML-based RPC. Jakarta XML Web Services offers extensive web
 services functionality, with support for multiple bindings/protocols.
-JAX-WS and JAX-RPC are fully interoperable when using the SOAP 1.1 over
+Jakarta XML Web Services and Jakarta XML-based RPC are fully interoperable when using the SOAP 1.1 over
 HTTP protocol as constrained by the WS-I Basic Profile specification.
-Support for JAX-RPC has been made optional as of Java EE 7. See
-link:#a2331[See Pruned Java
+Support for Jakarta XML-based RPC has been made optional as of Java EE 7. See
+link:#a2331[See Pruned Jakarta
 Technologies]”.
 
 JAX-WS and the Java Architecture for XML
 Binding (JAXB) define the mapping between Java classes and XML as used
 in SOAP calls, and provide support for 100% of XML Schema. JAXB is
-included in Java SE. The SOAP with Attachments API for Java (SAAJ),
+included in Java SE. Jakarta SOAP with Attachments,
 which is also included in Java SE, provides support for manipulating low
-level SOAP messages. The Web Services for Java EE specification fully
+level SOAP messages. The Web Services for Jakarta EE specification fully
 defines the deployment of web service clients and web service endpoints
 in Java EE, as well as the implementation of web service endpoints using
-enterprise beans. The Web Services Metadata specification defines Java
+enterprise beans. The Jakarta Enterprise Web Services specification defines Java
```

Comment:
This is another thing that shouldn't be changed.

The Web Services Metadata spec isn't part of Jakarta EE 8.
(And Jakarta Enterprise Web Services is a completely different spec.)